packages/base/core/src/less/_variables.less

```diff
@@ -219,6 +219,15 @@
 @oui-choice-thumbnail-border-color_disabled: @oui-color-charlotte;
 @oui-choice-thumbnail-border-color_error: @oui-color-error-light;
 
+@oui-choice-thumbnail-footer-background: @oui-color-charlotte;
+@oui-choice-thumbnail-footer-background_disabled: fade(@oui-color-charlotte, 50%);
+@oui-choice-thumbnail-footer-font-weight: @oui-font-semibold;
+@oui-choice-thumbnail-footer-font-size: rem-calc(18);
+@oui-choice-thumbnail-footer-margin-top: rem-calc(10);
+@oui-choice-thumbnail-footer-margin-bottom: rem-calc(-20);
+@oui-choice-thumbnail-footer-margin-top-checked: rem-calc(9);
```

Comment:
We should calculte this value by using `@oui-choice-thumbnail-border-width` and `@oui-choice-thumbnail-border-width-checked`, instead of having the value hardcoded